Command/RouterDebugExposedCommand.php

```diff
@@ -23,18 +26,23 @@
  *
  * @author      William DURAND <william.durand1@gmail.com>
  */
-class RouterDebugExposedCommand extends RouterDebugCommand
+class RouterDebugExposedCommand extends ContainerAwareCommand
```

Comment:
`RouterDebugCommand` became `final` in Symfony 3.4. But we only extended it for Symfony 2.3 support. So, it was an easy win to stop extending it.